combat-trainer.lic

```diff
@@ -2243,6 +2244,7 @@ class AttackProcess
     game_state.selected_maneuver = charged_maneuver unless game_state.loaded
     game_state.loaded = false if game_state.mob_died && !(game_state.casting_consume || game_state.prepare_consume)
     game_state.clear_aim_queue if Flags['ct-ranged-ready']
+    game_state.loaded = false if Flags['ct-aim-failed']
```

Comment:
I think this should be game_state.aiming, not loaded

Reply:
There is no `aiming` method in GameState. There is a `done_aiming?` which is a check to see if the aiming action array is empty (which is triggered to be empty by the line above this one if the aim is complete. So, setting loaded to false here will cause another "load" which will then begin to aim. This is just simply how the method works for this.

Reply:
Does it try to re-load the weapon after a failure?

Reply:
Yes, that's just the way the system works as is. So, it will hit the last else in the if chain in the `attack_aimed` method. It will hit the "already loaded" and then go about the normal aiming, etc.